prowler/providers/aws/services/rds/rds_service.py

```diff
@@ -436,8 +434,8 @@ class DBCluster(BaseModel):
     auto_minor_version_upgrade: bool
     multi_az: bool
     parameter_group: str
-    force_ssl: Optional[bool]
-    require_secure_transport: Optional[str]
+    force_ssl: Optional[str] = "0"
+    require_secure_transport: Optional[str] = "OFF"
```

Comment:
Maybe the optional is not needed here since you have default values, is not going to be `None`.

Reply:
True!